sections/findify-autocomplete.liquid

```diff
@@ -0,0 +1,101 @@
+{%- assign page_url = content_for_header | split:'"pageurl":"' | last | split:'"' | first | split: request.host | last | replace:'\/','/' | replace:'%20',' ' | replace:'\u0026','&'  -%}
+
+{%- assign query = page_url | split: 'findify-autocomplete&q=' -%}
+{%- assign suggestionArray = query[1] | split: '&suggestion=' | first | split: 'isMobile' | first -%}
+{%- assign suggestions = suggestionArray | split: ',' -%}
```

Comment:
you can probably do it in 1 line.

And why are you splitting by isMobile as well?

you should be able to take suggestions doing:

split: 'suggestions=' | last | split: '|' | first